runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -1439,6 +1529,14 @@ class VM_JFRConstantPoolTypes {
 			goto done;
 		}
 
+		_nativeLibrariesTable = pool_new(sizeof(NativeLibraryEntry), 0, sizeof(U_64), 0, J9_GET_CALLSITE(), OMRMEM_CATEGORY_VM, POOL_FOR_PORT(privatePortLibrary));
+		if(NULL == _nativeLibrariesTable ) {
```

Comment:
missing space `if (cond) {`

Reply:
If you could fix lines 1516, 1522 and 1528 as well, that would be appreciated.